samples/test/samples.test.js

```diff
@@ -26,9 +27,22 @@ const compute = new Compute();
 describe('samples', () => {
   describe('quickstart', () => {
     const name = `gcloud-ubuntu-${uuid.v4().split('-')[0]}`;
+
     after(async () => deleteVM(name));
-    it('should run the quickstart', () => {
+    it('should run the quickstart', async () => {
       const output = execSync(`node quickstart ${name}`);
+      const externalIP = execSync(`gcloud compute instances describe ${name} \
```

Comment:
So Im not saying this is a bad idea, just that I don't think we've used this pattern anywhere else with our tests.  This adds a dependency where you have to have `gcloud` installed on your machine, which is something we've avoided to this stage.  I could see us accomplishing the same thing in 2 ways:
- Having the quickstart `console.log` something like `ip: <ip>`, and then using a regex to parse it out
- passing in an instance-name (I guess we do), and then using the `Compute` API to query for the IP

Adding @bcoe because I think it's important to get this pattern consistent

Reply:
Awesome! I'm open to ideas here. I actually used this in [nodejs-getting-started](https://github.com/GoogleCloudPlatform/nodejs-getting-started/blob/bfe10211c2aa43bf1b1e0af6a03f4a98b76bfd7b/gce/test/app.test.js#L59), but I'm also happy to change that one as well if we decide on something else.